pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/stats/SubscriptionStatsImpl.java

```diff
@@ -93,6 +93,9 @@ public class SubscriptionStatsImpl implements SubscriptionStats {
     /** Total rate of messages expired on this subscription (msg/s). */
     public double msgRateExpired;
 
+    /** The count of expired messages on this subscription. */
+    public long msgExpired;
```

Comment:
Is the counter behind `msgRateExpired` matching `msgExpired` or `totalMsgExpired`?